exercise 25/index.js

```diff
@@ -0,0 +1,39 @@
+let wn= 2;
+if (wn==1)
+console.log("today is m");
+else if (wn==2)
+console.log("today is t");
+else if (wn==3)
+console.log("today is w");
+else if (wn==4)
+console.log("today is t");
+else if (wn==5)
+console.log("today is f");
+else if (wn==6)
+console.log("today is sat");
+else if (wn==7)
+console.log("today is sun");
+
+else console.log("number is not between 1-7");
```

Comment:
When you write many case condition, it's better to use curly braces:
`if(condition1) {`
`//statement1`
`} else if (condition2) {`
`//statement2`
`} else {`
`//statement 3`
`}`